documentation/src/main/wiki/releases/2.0.0/Documentation-2.0.0.md

```diff
@@ -3410,6 +3410,9 @@ It is possible to search tests and URLs by the query. Searched fields are the UR
 * use **[** / **]** to navigate between test's urls
 * press **m** to show/hide layout mask (when available)
 * press **←** / **→** to navigate between url's tabs
+* press **a** for accepting suite/test/url
```

Comment:
Please update only `SuiteReportFeatures.md`. I think that it isn't a good idea to update historical documentation ;)